### PR TITLE
add a format.ProcessTSV that works like ProcessJSON on TSV type messages

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: format/processtsv.go [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+					values[directive.index].value = timestamp.Format(directive.parameters[1])
+				}
+			}
+
+		case "replace":
+			if numParameters == 2 {
+				values[directive.index].value = strings.Replace(value, shared.Unescape(directive.parameters[0]), shared.Unescape(directive.parameters[1]), -1)
+			}
+
+		case "prefix":
```
Comment (arnecls) [inline format/processtsv.go]: please add this to the formatter documentation

File: format/processtsv.go [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+			if numParameters == 1 {
+				values[directive.index].value = shared.Unescape(directive.parameters[0]) + value
+			}
+
+		case "postfix":
```
Comment (arnecls) [inline format/processtsv.go]: please add this to the formatter documentation

File: format/processtsv.go [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+
+		case "trim":
+			switch {
+			case numParameters == 0:
+				values[directive.index].value = strings.Trim(value, " ")
+			case numParameters == 1:
+				values[directive.index].value = strings.Trim(value, shared.Unescape(directive.parameters[0]))
+			}
+
+		case "quote":
```
Comment (arnecls) [inline format/processtsv.go]: please add this to the formatter documentation

**Comment by arnecls** [comment]
Could you add a unit test for this, too?
This will make it easer to transfer it to the 0.5.0 branch.

**Comment by relud** [comment]
will add unit test

**Comment by arnecls** [comment]
Changes are fine for me.
I'll merge this to make it part of the 0.4.4 release.
Please do a separate pull request for the unit test.

And thanks again for your contributions :+1:

**Pull request merged**